2021q3/wg.adoc

```diff
@@ -0,0 +1,19 @@
+=== WireGuard on FreeBSD Stabilizes, Eyes Upstreaming
+
+Links +
+link:https://www.wireguard.com/[WireGuard] URL:link:https://www.wireguard.com/[https://www.wireguard.com/] +
+link:https://git.zx2c4.com/wireguard-freebsd/[wireguard-freebsd codebase] URL:link:https://git.zx2c4.com/wireguard-freebsd/[https://git.zx2c4.com/wireguard-freebsd/] +
+
+Contact: Jason A. Donenfeld <Jason@zx2c4.com>
+
+WireGuard is a secure tunneling protocol that lives in the kernel.
+
+For the last quarter, the https://git.zx2c4.com/wireguard-freebsd/[wireguard-freebsd codebase] has been quite stable and complete. For a while, there were rapid fire releases fixing issues, and a lot of effort was made to track down every bug report on bugs.freebsd.org, IRC, the mailing list, or otherwise. But by now, the reports have dried up, and mostly users come to IRC with questions on usage and integration, the usual types of things associated with a stabler project. We also have automated CI now for each commit, compiling and running a small smoke test on the repo's supported releases -- 12.2, and 13.0. At some point, hopefully that https://git.zx2c4.com/wireguard-freebsd/tree/tests/netns.sh[small smoke test] will expand to include the https://git.zx2c4.com/wireguard-linux/tree/tools/testing/selftests/wireguard/netns.sh[larger battery of tests] from Linux.
```

Comment:
"releases -- 12.2, and 13.0."

Not sure what happened exactly, but the document I emailed you said "12.1, 12.2, and 13.0".

While 12.1 isn't supported by FreeBSD anymore, the repo still has backports for it, because some people still use it.